src/main/java/robot/subsystems/elevator/commands/ElevatorCommand.java

```diff
@@ -0,0 +1,60 @@
+package robot.subsystems.elevator.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+import robot.subsystems.elevator.Constants;
+import robot.subsystems.elevator.Elevator;
+
+/**
+ *  Move the elevator to a certain height
+ */
+public class ElevatorCommand extends Command {
+    private double delta = 0.1;
+    private Elevator elevator = Robot.elevator;
+    private double height;
+
+    /**
+     * Make the elevator move to a specific height.
+     *
+     * @param height height in meters of the elevator
+     */
+    public ElevatorCommand(double height) {
+        requires(elevator);
+        this.height = height;
+    }
+
+    /**
+     * Make the elevator move to one of the predefined heights.
+     *
+     * @param state an enum of heights, defined in the elevator constants class.
+     */
+    public ElevatorCommand(Constants.ELEVATOR_STATES state) {
+        super(state.getLevelHeight());
```

Comment:
Did you mean using `this` instead?
```suggestion
        this(state.getLevelHeight());
```

Reply:
IIRC both do the same thing, but you are most likely right.

Reply:
@pauloalbert, I don't think that is the case. At least in Python, calling `super` inside a constructor will call the superclass' (parent class) constructor. So I believe `this` will call another constructor inside the same class, but calling `user` will call the constructor of `Command`, this isn't wanted in this case.
Actually, are you sure it even compiled? `Command`'s constructor doesn't get `int`s, I believe.

_That is a helluva throwback... I actually miss writing and reviewing FRC code. Hopefully the "reviewing" part will still happen. Good luck in the season, hopefully I'll be able to spend a lot of time with y'all._

###### The comment above shows weird things I think and write while I'm procrastinating at work